src/App.js

```diff
@@ -4,9 +4,19 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    inputValue: '',
+  };
 
   render() {
+    const { inputValue } = this.state;
+
+    const filmFilterOnChange
+      = [...moviesFromServer].filter(({ title, description }) => (
```

Comment:
```suggestion
      = moviesFromServer.filter(({ title, description }) => (
```
filter не меняет исходный массив